rasa/model.py

```diff
@@ -36,7 +36,7 @@
 
 
 # Type alias for the fingerprint
-Fingerprint = Dict[Text, Union[Text, List[Text], int, float]]
+Fingerprint = Dict[Text, Union[Optional[Text], List[Text], int, float]]
```

Comment:
I think it should be a TypedDict or a dataclass, not sure how hard it would be to make the change

Reply:
Should be fairly simple 👍